src/Assert.php

```diff
@@ -817,6 +818,24 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $expected, $message = '')
+    {
+        static::string($expected);
+
+        try {
+            $expression();
+        } catch (Exception $e) {
+            if (is_a($e, $expected, true)) {
+                return;
+            }
+        }
+
+        throw new InvalidArgumentException($message ?: sprintf(
+            'expected expression to throw an exception of type "%s"',
```

Comment:
Can you uppercase the first "Expected"? (consistency with other exception messages)